doc/cephfs/fs-nfs-exports.rst

```diff
@@ -17,29 +17,52 @@ Create NFS Ganesha Cluster
 
 .. code:: bash
 
-    $ ceph nfs cluster create <type=cephfs> [--size=1] <clusterid>
+    $ ceph nfs cluster create <type=cephfs> <clusterid> [<placement>]
 
 This creates a common recovery pool for all Ganesha daemons, new user based on
 cluster_id and common ganesha config rados object.
 
-Here size denotes the number of ganesha daemons within a cluster and type is
-export type. Currently only CephFS is supported.
+Here type is export type and placement specifies the size of cluster and hosts.
+For more details on placement specification refer the `orchestrator doc
+<https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification>`_.
+Currently only CephFS export type is supported.
 
-.. note:: This does not setup ganesha recovery database and start the daemons.
-          It needs to be done manually if not using vstart for creating
-          clusters. Please refer `ganesha-rados-grace doc
-          <https://github.com/nfs-ganesha/nfs-ganesha/blob/next/src/doc/man/ganesha-rados-grace.rst>`_
+Update NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster update <clusterid> <placement>
```

Comment:
`<placement>` means the cluster size, AFAICT? Let's just document that.

Reply:
> (btw, I for one would also be ok with splitting the cephadm integration and the teuthology tests)

We cannot split it. The teuthology tests require this PR.

Reply:
Along with cluster size, it is the [orch placement specification](https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification). I have added link about it in cluster create [here](https://github.com/ceph/ceph/pull/34672/files#diff-4b2d2894d071770202a68717833a3e53R27).